modules/cert-manager-acme-http01.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-http01_{context}"]
+= Configuring an ACME issuer to solve HTTP-01 challenges
+
+You can use {cert-manager-operator} to set up an ACME issuer to solve HTTPS01 challenges. This procedure uses _Let's Encrypt_ as the ACME CA server.
```

Comment:
I'm not sure whether "Let's Encrypt" should be italicized. I'll leave it up to you. AFAIK, we're supposed to italicize variables, words that are defined or explained, words that are identified as words, and occasionally, words that are emphasized. (Source: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-style.)